apisix/admin/init.lua

```diff
@@ -316,6 +330,11 @@ local uri_route = {
         methods = {"PUT"},
         handler = post_reload_plugins,
     },
+    {
+        paths = [[/apisix/admin/server_info]],
```

Comment:
I think we need do this in DP, not admin API

Reply:
And we should implement this feature by plugin way.

Take a look at: https://github.com/apache/apisix/blob/master/apisix/plugins/node-status.lua

Reply:
OK, will change it.

Reply:
I think the node-status plugin is suitable to also contains the server info.

Reply:
Fixed.

Reply:
@membphis @tokers 
The API should go to the control API: https://github.com/apache/apisix/issues/2798.
There is security risk if we put this feature into a plugin. Like the prometheus metrics, we don't have a default way to protect them.

Reply:
What about the control API's milestone?

Reply:
There is not milestone. So far, we can hide the entry first.